0-common/scala-core/vagrant/up.playbooks

```diff
@@ -0,0 +1,3 @@
+oss-playbooks/java7.yml
+oss-playbooks/scala.yml
+oss-playbooks/sbt.yml
```

Comment:
Are we sure we want the iglu-core subproject to have its own Vagrant startup logic?